packages/biotk/biotk.0.0.0/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+synopsis: "Bioinformatics toolkit"
+description: """
+biotk implements various functionalities in
+                computational biology, like parsers/unparsers for some
+                file formats, a few basic data structures and sequence
+                algorithms, as well as access to some databases."""
+maintainer: ["philippe.veber@gmail.com"]
+authors: ["Philippe Veber"]
+license: "CeCILL-B"
+homepage: "https://github.com/pveber/biotk/"
+bug-reports: "https://github.com/pveber/biotk/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "angstrom" {>= "0.15.0"}
+  "angstrom-unix" {>= "0.15.0"}
+  "biocaml"
+  "core" {>= "v0.14.0"}
+  "fmt"
+  "ocaml" {>= "4.11.0"}
+  "ocamlify"
+  "ocamlnet"
+  "otfm"
+  "ppx_csv_conv"
+  "ppx_deriving"
+  "result" {>= "1.5"}
+  "rresult" {>= "0.6.0"}
+  "vg"
+  "tyxml" {>= "4.4.0"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/pveber/biotk.git"
```

Comment:
```suggestion
dev-repo: "git+https://github.com/pveber/biotk.git"
available: arch != "x86_32" & arch != "arm32"
```

Due to
```
#=== ERROR while compiling biocaml.0.11.1 =====================================#
# context              2.1.0 | linux/arm32 | ocaml-base-compiler.4.12.0 | file:///src
# path                 ~/.opam/4.12/.opam-switch/build/biocaml.0.11.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p biocaml -j 31
# exit-code            1
# env-file             ~/.opam/log/biocaml-19-f59994.env
# output-file          ~/.opam/log/biocaml-19-f59994.out
### output ###
#       ocamlc lib/unix/.biocaml_unix.objs/byte/biocaml_unix__Bam.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.12/bin/ocamlc.opt -w -40 -short-paths -open Core_kernel -g -bin-annot -I lib/unix/.biocaml_unix.objs/byte -I /home/opam/.opam/4.12/lib/angstrom -I /home/opam/.opam/4.12/lib/base -I /home/opam/.opam/4.12/lib/base/base_internalhash_types -I /home/opam/.opam/4.12/lib/base/caml -I /home/opam/.opam/4.12/lib/base/md5 -I /home/opam/.opam/4.12/lib/base/shadow_stdlib -I /home/opam/.opam/4.12/lib/base64 -I /home/opam/.opam/4.12/lib/base_bigstring -I /home/opam/.opam/4.12/lib/base_quickcheck -I /home/opam/.opam/4.12/lib/bigarray-compat -I /home/opam/.opam/4.12/lib/bigstringaf -I /home/opam/.opam/4.12/lib/bin_prot -I /home/opam/.opam/4.12/lib/bin_prot/shape -I /home/opam/.opam/4.12/lib/bytes -I /home/opam/.opam/4.12/lib/cfstream -I /home/opam/.opam/4.12/lib/core_kernel -I /home/opam/.opam/4.12/lib/core_kernel/base_for_tests -I /home/opam/.opam/4.12/lib/core_kernel/binary_packing -I /home/opam/.opam/4.12/lib/fieldslib -I /home/opam/.opam/4.12/lib/jane-street-headers -I /home/opam/.opam/4.12/lib/parsexp -I /home/opam/.opam/4.12/lib/ppx_assert/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_bench/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_compare/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_enumerate/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_expect/collector -I /home/opam/.opam/4.12/lib/ppx_expect/common -I /home/opam/.opam/4.12/lib/ppx_expect/config -I /home/opam/.opam/4.12/lib/ppx_expect/config_types -I /home/opam/.opam/4.12/lib/ppx_expect/evaluator -I /home/opam/.opam/4.12/lib/ppx_expect/matcher -I /home/opam/.opam/4.12/lib/ppx_hash/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_here/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_inline_test/config -I /home/opam/.opam/4.12/lib/ppx_inline_test/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_module_timer/runtime -I /home/opam/.opam/4.12/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.12/lib/ppxlib/print_diff -I /home/opam/.opam/4.12/lib/re -I /home/opam/.opam/4.12/lib/re/perl -I /home/opam/.opam/4.12/lib/result -I /home/opam/.opam/4.12/lib/rresult -I /home/opam/.opam/4.12/lib/seq -I /home/opam/.opam/4.12/lib/sexplib -I /home/opam/.opam/4.12/lib/sexplib0 -I /home/opam/.opam/4.12/lib/splittable_random -I /home/opam/.opam/4.12/lib/stdio -I /home/opam/.opam/4.12/lib/stringext -I /home/opam/.opam/4.12/lib/time_now -I /home/opam/.opam/4.12/lib/typerep -I /home/opam/.opam/4.12/lib/uri -I /home/opam/.opam/4.12/lib/variantslib -I /home/opam/.opam/4.12/lib/xmlm -I /home/opam/.opam/4.12/lib/zip -I lib/base/.biocaml_base.objs/byte -intf-suffix .ml -no-alias-deps -open Biocaml_unix -o lib/unix/.biocaml_unix.objs/byte/biocaml_unix__Bam.cmo -c -impl lib/unix/bam.pp.ml)
# File "lib/unix/bam.ml", line 627, characters 13-23:
# 627 |       | Some 2147483647 (* POS in BAM is 0-based *)
#                    ^^^^^^^^^^
# Error: Integer literal exceeds the range of representable integers of type int
#       ocamlc lib/unix/.biocaml_unix.objs/byte/biocaml_unix__Sam.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.12/bin/ocamlc.opt -w -40 -short-paths -open Core_kernel -g -bin-annot -I lib/unix/.biocaml_unix.objs/byte -I /home/opam/.opam/4.12/lib/angstrom -I /home/opam/.opam/4.12/lib/base -I /home/opam/.opam/4.12/lib/base/base_internalhash_types -I /home/opam/.opam/4.12/lib/base/caml -I /home/opam/.opam/4.12/lib/base/md5 -I /home/opam/.opam/4.12/lib/base/shadow_stdlib -I /home/opam/.opam/4.12/lib/base64 -I /home/opam/.opam/4.12/lib/base_bigstring -I /home/opam/.opam/4.12/lib/base_quickcheck -I /home/opam/.opam/4.12/lib/bigarray-compat -I /home/opam/.opam/4.12/lib/bigstringaf -I /home/opam/.opam/4.12/lib/bin_prot -I /home/opam/.opam/4.12/lib/bin_prot/shape -I /home/opam/.opam/4.12/lib/bytes -I /home/opam/.opam/4.12/lib/cfstream -I /home/opam/.opam/4.12/lib/core_kernel -I /home/opam/.opam/4.12/lib/core_kernel/base_for_tests -I /home/opam/.opam/4.12/lib/core_kernel/binary_packing -I /home/opam/.opam/4.12/lib/fieldslib -I /home/opam/.opam/4.12/lib/jane-street-headers -I /home/opam/.opam/4.12/lib/parsexp -I /home/opam/.opam/4.12/lib/ppx_assert/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_bench/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_compare/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_enumerate/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_expect/collector -I /home/opam/.opam/4.12/lib/ppx_expect/common -I /home/opam/.opam/4.12/lib/ppx_expect/config -I /home/opam/.opam/4.12/lib/ppx_expect/config_types -I /home/opam/.opam/4.12/lib/ppx_expect/evaluator -I /home/opam/.opam/4.12/lib/ppx_expect/matcher -I /home/opam/.opam/4.12/lib/ppx_hash/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_here/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_inline_test/config -I /home/opam/.opam/4.12/lib/ppx_inline_test/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_module_timer/runtime -I /home/opam/.opam/4.12/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.12/lib/ppxlib/print_diff -I /home/opam/.opam/4.12/lib/re -I /home/opam/.opam/4.12/lib/re/perl -I /home/opam/.opam/4.12/lib/result -I /home/opam/.opam/4.12/lib/rresult -I /home/opam/.opam/4.12/lib/seq -I /home/opam/.opam/4.12/lib/sexplib -I /home/opam/.opam/4.12/lib/sexplib0 -I /home/opam/.opam/4.12/lib/splittable_random -I /home/opam/.opam/4.12/lib/stdio -I /home/opam/.opam/4.12/lib/stringext -I /home/opam/.opam/4.12/lib/time_now -I /home/opam/.opam/4.12/lib/typerep -I /home/opam/.opam/4.12/lib/uri -I /home/opam/.opam/4.12/lib/variantslib -I /home/opam/.opam/4.12/lib/xmlm -I /home/opam/.opam/4.12/lib/zip -I lib/base/.biocaml_base.objs/byte -intf-suffix .ml -no-alias-deps -open Biocaml_unix -o lib/unix/.biocaml_unix.objs/byte/biocaml_unix__Sam.cmo -c -impl lib/unix/sam.pp.ml)
# File "lib/unix/sam.ml", line 216, characters 34-44:
# 216 |   (if (1 <= length) && (length <= 2147483647) then
#                                         ^^^^^^^^^^
# Error: Integer literal exceeds the range of representable integers of type int
#     ocamlopt lib/unix/.biocaml_unix.objs/native/biocaml_unix__Sam.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.12/bin/ocamlopt.opt -w -40 -short-paths -open Core_kernel -g -I lib/unix/.biocaml_unix.objs/byte -I lib/unix/.biocaml_unix.objs/native -I /home/opam/.opam/4.12/lib/angstrom -I /home/opam/.opam/4.12/lib/base -I /home/opam/.opam/4.12/lib/base/base_internalhash_types -I /home/opam/.opam/4.12/lib/base/caml -I /home/opam/.opam/4.12/lib/base/md5 -I /home/opam/.opam/4.12/lib/base/shadow_stdlib -I /home/opam/.opam/4.12/lib/base64 -I /home/opam/.opam/4.12/lib/base_bigstring -I /home/opam/.opam/4.12/lib/base_quickcheck -I /home/opam/.opam/4.12/lib/bigarray-compat -I /home/opam/.opam/4.12/lib/bigstringaf -I /home/opam/.opam/4.12/lib/bin_prot -I /home/opam/.opam/4.12/lib/bin_prot/shape -I /home/opam/.opam/4.12/lib/bytes -I /home/opam/.opam/4.12/lib/cfstream -I /home/opam/.opam/4.12/lib/core_kernel -I /home/opam/.opam/4.12/lib/core_kernel/base_for_tests -I /home/opam/.opam/4.12/lib/core_kernel/binary_packing -I /home/opam/.opam/4.12/lib/fieldslib -I /home/opam/.opam/4.12/lib/jane-street-headers -I /home/opam/.opam/4.12/lib/parsexp -I /home/opam/.opam/4.12/lib/ppx_assert/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_bench/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_compare/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_enumerate/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_expect/collector -I /home/opam/.opam/4.12/lib/ppx_expect/common -I /home/opam/.opam/4.12/lib/ppx_expect/config -I /home/opam/.opam/4.12/lib/ppx_expect/config_types -I /home/opam/.opam/4.12/lib/ppx_expect/evaluator -I /home/opam/.opam/4.12/lib/ppx_expect/matcher -I /home/opam/.opam/4.12/lib/ppx_hash/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_here/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_inline_test/config -I /home/opam/.opam/4.12/lib/ppx_inline_test/runtime-lib -I /home/opam/.opam/4.12/lib/ppx_module_timer/runtime -I /home/opam/.opam/4.12/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/4.12/lib/ppxlib/print_diff -I /home/opam/.opam/4.12/lib/re -I /home/opam/.opam/4.12/lib/re/perl -I /home/opam/.opam/4.12/lib/result -I /home/opam/.opam/4.12/lib/rresult -I /home/opam/.opam/4.12/lib/seq -I /home/opam/.opam/4.12/lib/sexplib -I /home/opam/.opam/4.12/lib/sexplib0 -I /home/opam/.opam/4.12/lib/splittable_random -I /home/opam/.opam/4.12/lib/stdio -I /home/opam/.opam/4.12/lib/stringext -I /home/opam/.opam/4.12/lib/time_now -I /home/opam/.opam/4.12/lib/typerep -I /home/opam/.opam/4.12/lib/uri -I /home/opam/.opam/4.12/lib/variantslib -I /home/opam/.opam/4.12/lib/xmlm -I /home/opam/.opam/4.12/lib/zip -I lib/base/.biocaml_base.objs/byte -I lib/base/.biocaml_base.objs/native -intf-suffix .ml -no-alias-deps -open Biocaml_unix -o lib/unix/.biocaml_unix.objs/native/biocaml_unix__Sam.cmx -c -impl lib/unix/sam.pp.ml)
# File "lib/unix/sam.ml", line 216, characters 34-44:
# 216 |   (if (1 <= length) && (length <= 2147483647) then
#                                         ^^^^^^^^^^
# Error: Integer literal exceeds the range of representable integers of type int
```

Reply:
shouldn't I add these constraints to biocaml instead, and inherit them via the dep wrt biocaml?

Reply:
Oh sorry, my bad, yes!

Reply:
I will do that in a separate PR